tests/unit_tests/test_tethys_apps/test_cli/test_install_commands.py

```diff
@@ -3,7 +3,7 @@
 from django.core.exceptions import ValidationError
 from unittest import mock
 from conda.cli.python_api import Commands
-from tethys_apps.cli import install_commands
+from tethys_cli import install_commands
```

Comment:
Wait, what?! How was this even working before? I think we just need to delete this file. It looks like it was replaced by `tests/unit_tests/test_tethys_cli/test_install_commands.py`.